README.md

```diff
@@ -1,43 +1,31 @@
 # Rancher Logging Examples
 
-This repository contains 2 example charts for testing your logging setup and a simple application for generating logs.
+This repository contains an example charts for testing your logging setup.
 
 ## Charts
 
-### Simple Application
-The `simple-application` chart will deploy 2 pods that will constantly out logs.  This chart is recommended for when you want to configure your own logging setup and need a simple application to collect logs from.
-
 ### Rancher Logging Example
-The `rancher-logging-example` is an end to end setup of an application, multiple configurations for log collection, and an elasticsearch cluster as an output.  This chart is recommended if you want to see a working example of the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart.  
+The `rancher-logging-example` is an end to end setup of an application, multiple configurations for log collection, and a simple output.  This chart is recommended if you want to see a working example of the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart.  

```

Comment:
nit: I think we could make the second sentence...
`This chart is recommended for running an example payload for the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart`.